src/uu/mknod/src/mknod.rs

```diff
@@ -5,22 +5,42 @@
 // For the full copyright and license information, please view the LICENSE
 // file that was distributed with this source code.
 
-// spell-checker:ignore (ToDO) parsemode makedev sysmacros makenod newmode perror IFBLK IFCHR IFIFO
-
-mod parsemode;
+// spell-checker:ignore (ToDO) parsemode makedev sysmacros makenod perror IFBLK IFCHR IFIFO
 
 #[macro_use]
 extern crate uucore;
 
+use std::ffi::CString;
+
+use clap::{App, Arg, ArgMatches};
 use libc::{dev_t, mode_t};
 use libc::{S_IFBLK, S_IFCHR, S_IFIFO, S_IRGRP, S_IROTH, S_IRUSR, S_IWGRP, S_IWOTH, S_IWUSR};
 
-use getopts::Options;
-
-use std::ffi::CString;
+mod parsemode;
 
 static NAME: &str = "mknod";
 static VERSION: &str = env!("CARGO_PKG_VERSION");
+static USAGE: &str = "
+Usage: mknod [OPTION]... NAME TYPE [MAJOR MINOR]
```

Comment:
I think you can split this to USAGE and HELP, then you can use `app.usage(USAGE)`, `app.after_help(LONG_HELP)`

Reply:
I have tried multiple ways, but this doesn't seem to work.
- use `usage` and `after_help` (as suggested): clap still adds the auto-generated help text duplicating what we specified in `after_help`
- use `usage` and `help`: usage is not shown at all

The alternative would be to move to clap's automatic help message entirely. I'm not sure how much we want to ensure that the formatting of GNU mknod's help message is retained, but if you want me to change it, please let me know.

Reply:
Sorry if my initial comment wasn't clear, i was suggesting something like this:

```rust
static ABOUT: &str = "Create the special file NAME of the given TYPE.";
static USAGE &str = "Usage: mknod [OPTION]... NAME TYPE [MAJOR MINOR]";
static LONG_HELP: &str = "
Both MAJOR and MINOR must be specified when TYPE is b, c, or u, and they
must be omitted when TYPE is p.  If MAJOR or MINOR begins with 0x or 0X,
it is interpreted as hexadecimal; otherwise, if it begins with 0, as octal;
otherwise, as decimal.  TYPE may be:

  b      create a block (buffered) special file
  c, u   create a character (unbuffered) special file
  p      create a FIFO

NOTE: your shell may have its own version of mknod, which usually supersedes
the version described here.  Please refer to your shell's documentation
for details about the options it supports.

GNU coreutils online help: <https://www.gnu.org/software/coreutils/>
Full documentation <https://www.gnu.org/software/coreutils/mknod>
or available locally via: info '(coreutils) mknod invocation'
";

pub fn uumain(args: impl uucore::Args) {
    App::new().usage(USAGE).after_help(LONG_HELP).about(ABOUT)
}
```

Reply:
Ah I see, thanks :+1: 

I have split it and added a small phrase to each of the arguments.